tests/utils.py

```diff
@@ -65,6 +65,7 @@ def module_v2_venv(module_path: str) -> Iterator[env.VirtualEnv]:
                 "-X",
                 "module",
                 "install",
+                "-e",
```

Comment:
This change scares me somewhat. There are subtle differences between the normal install and the editable one that are mostly irrelevant in normal operation but that cause tricky issues when done in-process. I need to give this some more thought. I do see why you do it of course, I guess we'll probably have to, but we do need to consider potential implications. Then again, if the tests succeed it's probably fine. I'll get back to you on this.

Reply:
Let's run with it, I think it's fine.